job-descriptions/software-engineer.md

```diff
@@ -4,10 +4,12 @@
 
 ### About the company
 
-From sustainable energy systems to self-driving cars, the innovations of the future will all rely on code. With so much software to write, developers need a better way to access, understand, and collaborate on code.
+At Sourcegraph, we are building a better, smarter foundation for software development. The innovations of the future will all rely on code and by empowering software developers today we can literally bring the future sooner.
 
-At Sourcegraph, we are building a better, smarter foundation for software development. One that will enable developers (and the companies that rely on them) to more quickly turn great ideas into groundbreaking technology. Effectively, we want to bring the future sooner.
-Check out what we are building for developers and why it matters: https://sourcegraph.com/plan.
+We have already built the world's best [code search](https://about.sourcegraph.com/docs/search/) that is used and loved by large and small engineering organizations across the world.
+[Sourcegraph's browser extensions](https://about.sourcegraph.com/product/browser) provide IDE-like code intelligence (e.g. hover tooltips, go to definition, find references, and much more) in your code host, in code review, and everywhere you read code.

```

Comment:
There's this weird thing here where we imply that Sourcegraph (the server/webapp) is for code search, while the browser extensions are for code intelligence. That's why I wanted both to be in the subject of this sentence. What do you think?